app/views/idv/in_person/ready_to_verify/show.html.erb

```diff
@@ -111,3 +111,7 @@
     </address>
   </section>
 <% end %>
+
+<%= render PageFooterComponent.new do %>
+  <%= link_to t('in_person_proofing.body.barcode.cancel_link_text'), idv_cancel_path(step: 'verify') %>
+<% end %>
```

Comment:
Yeah, the reason I didn't use those is that they use a different text string for the link text ("Cancel" instead of "Cancel your barcode"). I considered adding a link text parameter to one of those but it didn't seem worth it since we're only doing this in one place and it doesn't seem likely that we'll want to do it elsewhere with different text again